docs/accounts/tenant/workspace-settings.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Workspaces
+sidebar_label: Workspaces
+---
+
+# Workspaces
+
+The **Workspaces** page allows you to control various settings which applies to all workspaces across the tenant.
+
+## Snapshot Visibility
+
+Snapshot Visibility setting allows you to control the permitted snapshot visibility options across all workspaces in your tenant.
+
+The following snapshot visibility settings are supported:
+
+| Setting      | Description                                                                                                                       |
+| ------------ | --------------------------------------------------------------------------------------------------------------------------------- |
+| **Workspace** | The snapshot is accessible to all members of the workspace in which the snapshot is created.                                     |
+| **Anyone with link** | The snapshot is visible to anyone who has the link.                                                                       |
+
+Refer [here](/pipes/docs/using/steampipe/snapshots#sharing-snapshots) for further information.
```

Comment:
I dont like to use "here" in the link.  Better just add the link to the sentence above - eg `The following [snapshot visibility](/pipes/docs/using/steampipe/snapshots#sharing-snapshots) settings are supported`

Reply:
Done